ivy/array/linear_algebra.py

```diff
@@ -185,10 +185,41 @@ def matrix_norm(
         /,
         *,
         ord: Optional[Union[int, float, Literal[inf, -inf, "fro", "nuc"]]] = "fro",
-        axis: Optional[Union[int, Sequence[int]]] = None,
+        axis: Optional[Tuple[int, int]] = (-2, -1),
```

Comment:
I don't know why this type hint change still appears eventhough it was already updated in this [PR](#5420)

Reply:
#5420

Reply:
did you pull the latest commit in this PR?

Reply:
done! now only shows the expected changes.